### ISSUE TITLE
Small Usability Improvement: New Command: Add Task to Task

### ISSUE THREAD

**Issue by schittli** [kanboard/forum#111]
Good evening
Thank you very much for your great work!, kanboard is making work much easier :-)

I have a small usability improvement proposal:
We never use subtasks, because of the missing properties and because they can not be individually assigned to Board columns. Therefore, 
### it would be useful, if kanboard could offer a "Add Task to Task" command, 
so that a Task must not be manually created and then manually assigned to a parent.

#### I guess the "Add Task to Task" command would be useful here:
- As an action in TaskViewController&action=show
- It would be great if the Task-Boxes in the Board could offer a "+" to add and assign Tasks, too: Most often if I study the Board, I detect missing Tasks :-)
- In TaskListController&action=show, it would be useful, too...

Thanks a lot, kind regards,
Thomas

**Comment by creecros** [comment]
I do not understand. Better explanation?

**Comment by creecros** [comment]
Is this like add a task the "relates to" a task?

**Comment by schittli** [comment]
Thank you for your question. Today, we can do the following:
1. Create a new Task 
2. Enter the data for the new Task and save it
2. Open the newly created Task with all details (in TaskViewController&action=show)
3. Select "Add internal link"
4. Select: Label: is a Child of
5. Select: Task: … search for the parent Task
6. Save
7. Go back to the website where we have started with step 1

My idea is to have a command which does all steps by one click, so that the User can:
1. Select "+ assigned Task" to create and assign a new Task to an existing Task
2. Enter the data for the new Task
3.  … and save it - done.

**Comment by creecros** [comment]
Yes, gotcha. I do this too, only I use "relates to". I will most likely develop a plug in for this, just started something else though...so may be awhile til I get to it, unless I give up on my next venture, which is possible since it's my second attempt.
